great_expectations/data_context/util.py

```diff
@@ -143,7 +144,9 @@ def format_dict_for_error_message(dict_):
     return "\n\t".join("\t\t".join((str(key), str(dict_[key]))) for key in dict_)
 
 
-def file_relative_path(dunderfile, relative_path):
+def file_relative_path(
+    source_path: str, relative_path: str, strict: bool = True
```

Comment:
https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/alias_types.py#L8
```suggestion
    source_path: PathStr, relative_path: PathStr, strict: bool = True
```